nvblox_ros/config/parameters.yaml

```diff
@@ -1,40 +1,175 @@
-use_helper_frame: true # CRITICAL: In the current setup, cam4_sensor_frame_helper should be used.
-global_frame: map
-pose_frame: base
-memory_checking_period: 5 # seconds
-max_meshing_height: 4 # meters
-memory_purge_threshold: 1000 # in MB,
-output_dir: "" # By default saves to "home/<username>/maps" folder
-voxel_size: 0.08 #0.05
-slice_height: 0.75
-min_height: 0.05
-max_height: 2
-esdf: false
-esdf_2d: false
-mesh: true
-distance_slice: false
-tsdf_integrator_max_integration_distance_m: 30.0
-tsdf_integrator_truncation_distance_vox: 4.0 # when increase it populates faster but so noisy
-tsdf_integrator_max_weight: 800.0 #higher is better for high quality
-color_integrator_max_integration_distance_m: 30.0
-esdf_integrator_min_weight: 2.0
-esdf_integrator_max_distance_m: 10.0
-esdf_integrator_min_site_distance_vox: 1.0
-max_tsdf_update_hz: 0.0
-max_color_update_hz: 0.0
-max_mesh_update_hz: 0.0
-max_esdf_update_hz: 2.0
-mesh_integrator_min_weight: 0.00001
-mesh_integrator_weld_vertices: false
+##########################
+### General Parameters ###
+##########################
+
+#Whether to use the static occupancy layer for projective integration. If this flag is set to false (default), TSDF integration is used.
+use_static_occupancy_layer: false
+
+#Voxel size (in meters) to use for the map.
+voxel_size: 0.05
+
+#The name of the TF frame to be used as the global frame.
+global_frame: "odom"
+
+#Only used if `use_topic_transforms` is set to true. Pose and transform messages will be interpreted as being in this pose frame, and the remaining transform to the sensor frame will be looked up on the TF tree.
+pose_frame: "base_link"
+
+# Depth Image Topic Name
+depth_image_topic_name: "/depth/image"
```

Comment:
I strongly prefer to have these remapped as normal topics rather than parameters, which just adds to the confusion. We had to do this in the ROS2 version for stupid ROS2 or Isaac ROS reasons, but I don't see much need to do this in ROS1.

Reply:
I'll change this in the next PR. :)